test/ActiveLogin.Authentication.BankId.AspNetCore.Test/Helpers/FakeRemoteIpAddressMiddleware.cs

```diff
@@ -0,0 +1,23 @@
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore.Test.Helpers
+{
+    internal class FakeRemoteIpAddressMiddleware
+    {
+        private readonly RequestDelegate next;
+        private readonly IPAddress fakeAddress = IPAddress.Parse("10.0.0.1");
+
+        public FakeRemoteIpAddressMiddleware(RequestDelegate next)
```

Comment:
I would propose to take the IPAddress as a parameter, then it will be very clear form the what what IP is used.
I had to read up a bit, but here is how it can be used.

Final code would be:
```csharp
    internal class FakeRemoteIpAddressMiddleware
    {
        private readonly RequestDelegate _next;
        private readonly IPAddress _fakeAddress;

        public FakeRemoteIpAddressMiddleware(RequestDelegate next, IPAddress fakeAddress)
        {
            _next = next;
            _fakeAddress = fakeAddress;
        }
...
}
```

And usage is:
```csharp
app.UseMiddleware<FakeRemoteIpAddressMiddleware>(IPAddress.Parse("192.0.2.1"));
```

Also, tried reading up one what IP to use for test. ``192.0.2.X` seems to be a good range :)
https://stackoverflow.com/a/10456065

Reply:
Nice! I added a comment as well in the code to refer to the RFC.